common/Resources/ti.internal/extensions/ti/ti.ui.js

```diff
@@ -32,6 +34,20 @@ Object.defineProperty(uiModule, 'semanticColorType', {
 	}
 });
 
+const hexToRgb = function hexToRgb(hex) {
+	let alpha = 1;
+	let color = hex;
+	if (hex.color) {
+	  alpha = hex.alpha / 100; // convert from 0-100 range to 0-1 range
```

Comment:
<!--
  1 failure:  common/Resources/...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/extensions/ti/ti.ui.js;
  Line: 41;
-->
- :no_entry_sign: common/Resources/ti.internal/extensions/ti/ti.ui.js line 41 – Mixed spaces and tabs. (no-mixed-spaces-and-tabs)